plugins/00_dokku-standard/subcommands/run

```diff
@@ -23,6 +24,16 @@ dokku_run_cmd() {
   [[ "$DOKKU_RM_CONTAINER" ]] && local DOKKU_RUN_OPTS="--rm"
   has_tty && local DOKKU_RUN_OPTS+=" -i -t"
   is_image_herokuish_based "$IMAGE" && local EXEC_CMD="/exec"
+
+  # TODO: should we cleanup the procfile after run?
+  extract_procfile "$APP"
```

Comment:
@josegonzalez @michaelshobbs I noticed that Procfiles are cleaned up after deployment. To be able to read the Procfile on `dokku run` I had to extract it first. Is this desirable? Unfortunately it yields a log message which is a bit distracting: `-----> App Procfile file found (/home/dokku/nodejs/DOKKU_PROCFILE)`.